proxy/src/outbound.rs

```diff
@@ -139,19 +136,18 @@ where
     /// changed.
     fn bind_service(
         &mut self,
-        key: &Self::Key,
+        dst: &Self::Key,
     ) -> Result<Self::Service, Self::RouteError> {
-        let &(ref dest, ref protocol) = key;
-        debug!("building outbound {:?} client to {:?}", protocol, dest);
+        debug!("building outbound client to {:?}", dst);
```

Comment:
TIOLI, really minor nit, but I think the original log line, which would say something like "building outbound Http1 client to 127.0.0.1:80" was probably more readable than the new log line, which might say things like "building outbound Http1 client to Dst::Unbound("127.0.0.1:80", Http1)". This probably doesn't really matter at all since it's a debug log line...